src/web/mjs/connectors/MangaFire.mjs

```diff
@@ -101,15 +101,16 @@ export default class MangaFire extends Connector {
                 const data = await this.fetchJSON(request);
                 const dom = this.createDOM(data.result.html);
                 const chaptersNodes = [...dom.querySelectorAll('a')];
-                chaptersNodes.map(chapter => {
-                    const id = {itemid : chapter.dataset.id, itemtype : type};
-                    const title = chapter.text.trim();
-                    chapterList.push ({
-                        id : JSON.stringify(id),
-                        title : title,
-                        language : language
+                chaptersNodes.filter(anchor=> anchor.pathname.includes(`/${type}-`))
+                    .map(chapter => {
```

Comment:
Recommend to use `forEach` instead of `map` to express the intend (it updates `chapterList` by pushing within the callback instead of pushing the result from `map`).